test/integration-test.py

```diff
@@ -64,39 +66,48 @@ def main():
     # allow any ACME client to run custom command for integration
     # testing (without having to implement its own busy-wait loop)
     parser.add_argument('--custom', metavar="CMD", help="run custom command")
-    parser.set_defaults(run_chisel=False, test_case_filter="", skip_setup=False)
+    parser.set_defaults(run_chisel=False, test_case_filter="", skip_setup=False, coverage=False, coverage_dir=None)
     args = parser.parse_args()
 
+    if args.coverage and not args.coverage_dir:
+        raise(Exception("must specify --coveragedir when using --coverage"))
```

Comment:
As noted above, you could easily have a default coveragedir here. Why don't you?